manual/src/refman/typedecl.etex

```diff
@@ -45,6 +45,8 @@ ext-variance:
 variance:
           '+'
         | '-'
+	| '+-'
+	| '-+'
```

Comment:
This could be vertically aligned with the `-` case.

Reply:
Argh, they are TAB characters. I will replace them by spaces in the next commit.